assets/css/admin.css

```diff
@@ -6,3 +6,7 @@ input#rsa-user-message {
 .rsa-user-message {
 	text-align: center
 }
+
+.rsa-ip-addresses-field-wrapper {
+	position: relative;
+}
```

Comment:
**Note for the reviewer**

This is added so that jQuery's shake effect works like before.